src/app/components/button/button.component.html

```diff
@@ -1 +1,13 @@
-<button>Next Step</button>
+<div>
+  @if (previousRoute) {
+  <button class="prev-button" (click)="goBack()">Go Back</button>
+  }
+  <button
+    class="next-button"
+    type="submit"
+    (click)="nextRoute !== '/step2' ? nextStep() : ''"
```

Comment:
avoid route names like step2. something like step-two is more appropriate

Reply:
Ok